documentation/roadmap/roadmap-v5.md

```diff
@@ -0,0 +1,41 @@
+# Grafana Operator v5.0 - Roadmap
+
+A list of features and changes for the 5.0 release of the Grafana Operator. The purpose of this document is to allow users to discuss and extend the proposed changes.
+
+## Multi Namespace Support
+
+The Operator should be capable of managing multiple Grafana instances in multiple namespaces. It is no longer a requirement that the Grafana instance is deployed to the same namespace as the Operator.
+
+Currently being worked on? Yes: [#599](https://github.com/grafana-operator/grafana-operator/pull/599)
+
+## Reconciler Update
+
+We want to switch to use `controllerutils.CreateOrUpdate` instead of manually checking the current cluster state in every reconciliation. This will reduce both, code size and Kubernetes API requests.
+
+Currently being worked on? Yes. No PR yet.
+
+## CRD version
+
+The CRD version will be updated to v1beta1. The group will change from `integreatly.org` to `grafana-operator`.
```

Comment:
Should we use grafana-operator. Maybe try to use a more generic but naming is hard :)